### ISSUE TITLE
Issue with deployed Remix contract

### ISSUE THREAD

**Issue by Prker** [HashLips/example_nft_minter#5]
2 days ago I could take the this smart contract and deploy it on Remix and was able to mint from both the owners address and other wallets. Last night I started to get an error that would not let any wallets, other then the owners, use the mint function. It seems that everything fails unless line 46 (in SmartContract.sol) is disabled. Anyone else having this issue? It seems like msg.value is passing as empty (or 0), therefor the transaction fails on line 46. 

I was able to deploy the contract to Rinkeby then verify the contract on etherscan and use the mint function that because Payable the payableAmount field and if filled out correctly the mint from the non-owner wallet works. 

New to Solidity so a bit stuck, any thoughts ?

**Comment by rkingbon** [comment]
Same here, started to get an error that would not let any wallets, other then the owners, use the mint function.

**Comment by abhay-rana** [comment]
yeah same error only the owner can run the mint function , when another wallet try to mint function it gives the gas estimation error and transaction reverted back.
plz solve this issue if anyone know how to solve this

**Comment by MindpowerLLC** [comment]
You should post your contract so people can see if there is an issue.
